docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,484 @@
+```
+The complete nonAdminBackup resource definition can be found here: [NonAdminBackup CRD](https://github.com/openshift/oadp-operator/blob/master/bundle/manifests/oadp.openshift.io_nonadminbackups.yaml)
+
+### NonAdminRestore NAR Example:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminRestore
+metadata:
+  name: example
+  namespace: nacuser1
+spec: 
+  restoreSpec:
+    backupName: mybackup-1
+```
+
+Once created the NAR will look similar to the following:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminRestore
+metadata:
+  creationTimestamp: "2025-02-21T21:12:54Z"
+  finalizers:
+  - nonadminrestore.oadp.openshift.io/finalizer
+  generation: 2
+  name: example
+  namespace: nacuser1
+  resourceVersion: "20719136"
+  uid: 0f1d8346-d8be-4621-8d67-0877f15e82fb
+spec:
+  restoreSpec:
+    backupName: mybackup-1
+    hooks: {}
+    itemOperationTimeout: 0s
+status:
+  conditions:
+  - lastTransitionTime: "2025-02-21T21:12:54Z"
+    message: restore accepted  <--- The NAC controller reconciled the NAR object and created the Velero Restore object
+    reason: RestoreAccepted
+    status: "True"
+    type: Accepted
+  - lastTransitionTime: "2025-02-21T21:12:54Z"
+    message: Created Velero Restore object  <--- The NAC controller created the Velero Restore object
+    reason: RestoreScheduled
+    status: "True"
+    type: Queued
+  phase: Created  <--- The NAR object is in the Created phase
+  queueInfo:
+    estimatedQueuePosition: 0  <--- The NAR object status in the queue, once complete it is set to 0
+  veleroRestore:
+    nacuuid: nacuser1-example-b844be97-7ee4-4702-91b8-ffc84697675a  <--- The NAC controller created the Velero Restore object and set the nacuuid
+    name: nacuser1-example-b844be97-7ee4-4702-91b8-ffc84697675a  <--- The associated Velero Restore name
+    namespace: openshift-adp
+    status:
+      completionTimestamp: "2025-02-21T21:12:57Z"
+      hookStatus: {}
+      phase: Completed  <--- The Velero restore object is in the Completed phase, successful
+      progress:
+        itemsRestored: 54
+        totalItems: 54
+      startTimestamp: "2025-02-21T21:12:55Z"
+      warnings: 0
+```
+The complete nonAdminRestore resource definition can be found here: [NonAdminRestore CRD](https://github.com/openshift/oadp-operator/blob/master/bundle/manifests/oadp.openshift.io_nonadminrestores.yaml)
+
+### NonAdminBackupStorageLocation NABSL:
+Cluster administrators can gain efficiencies by delegating backup and restore operations to OpenShift users. It is recommended that cluster administrators carefully manage the NABSL to conform to any company policies, compliance requirements, etc.  
+
+1. **Direct Creation**: Cluster administrators can create NABSLs directly for non-admin users.
+2. **Approval Workflow**: Cluster administrators can enable an approval process where:
+   - Users create NABSL's from the user namespace, this creates a NABSLApprovalRequest object in the openshift-adp namespace.
```

Comment:
It's not NABSLApprovalRequest it's NABSLRequest :) `NonAdminBackupStorageLocationRequest`

Reply:
- Users create a NABSL, which triggers the creation of a NABSLRequest object in the openshift-adp namespace.
- Administrators review and either approve or reject these requests. Once approved, a Velero BSL is created in the openshift-adp namespace, and the user is notified of the approval on the NABSL status. If rejected, the status of the NABSL is updated to reflect the rejection.
- Administrators can also revoke previously approved NABSL, which results in the removal of the Velero BSL, and the user is notified of the rejection. This is achieved by modifying the approve field back to pending or reject.

Reply:
Update of NSL is not supported.